src/trading/tradingSdk.ts

```diff
@@ -72,6 +73,13 @@ export class TradingSdk {
     return getQuoteWithSigner(this.mergeParams(params), advancedSettings, this.options.orderBookApi)
   }
 
+  async getSuggestedSlippageBps(
+    params: WithPartialTraderParams<TradeParameters>,
+    advancedSettings?: SwapAdvancedSettings
+  ): Promise<number> {
+    return getSuggestedSlippageBpsWithSigner(this.mergeParams(params), advancedSettings, this.options.orderBookApi)
```

Comment:
`getSuggestedSlippageBpsWithSigner` makes a `quote` API request which contains a lot of data, but here we only return slippage. Maybe we should return quote AND suggestedSlippage together?
This comment is related to https://github.com/cowprotocol/cow-sdk/pull/306/files#r2086574195.